### ISSUE TITLE
Add example for translation from GAPDoc to AutoDoc

### ISSUE THREAD

**Issue by sebasguts** [gap-packages/AutoDoc#134]
Take an example from the GAP library where doku is written in GAPDoc, and translate it to AutoDoc

**Comment by fingolfin** [comment]
Yeah, that would be nice; it would also highlight how much shorter things can become. It would fit well into a new subsection at the end of section 1.3 in the manual, I guess.

This is an issue well-suited for new contributors to AutoDoc, too.
